output/cloud/expv2/sink_test.go

```diff
@@ -16,7 +17,14 @@ func TestNewSink(t *testing.T) {
 		{metrics.Counter, &counter{}},
 		{metrics.Gauge, &gauge{}},
 		{metrics.Rate, &rate{}},
-		{metrics.Trend, &histogram{}},
+		{
+			metrics.Trend,
+			&histogram{
```

Comment:
newHistogram maybe :sweat_smile:

Reply:
I was a bit unsure about it, but thinking more about it, it makes sense.